src/cfi_backward.adoc

```diff
@@ -98,8 +98,9 @@ The Zicfiss extension introduces the following instructions:
 * Perform an atomic swap from a shadow stack location (See <<SSAMOSWAP>>)
 ** `SSAMOSWAP.W` and `SSAMOSWAP.D`
 
-When a `MOP.RR.7` or `MOP.R.28` encoding is not utilized by the Zicfiss
-extension, the instruction adheres to its Zimop-defined behavior, unless it is
+Zicfiss does not use all encodings of `MOP.RR.7` or `MOP.R.28`. When a
+`MOP.RR.7` or `MOP.R.28` encoding is not utilized by the Zicfiss extension, the
+corresponding instruction adheres to its Zimop-defined behavior, unless it is
 employed by another extension. In such cases, the instruction follows the
 behavior specified by that other extension.
```

Comment:
I suggest shortening `unless it is employed by another extension. In such cases, the instruction follows the behavior specified by that other extension.` to `unless redefined by another extension.`  This matches the language in the Zimop spec and avoids making it seem like this is some kind of special case.

Reply:
@aswaterman updated. Thanks!